opentelemetry-api/src/opentelemetry/trace/span.py

```diff
@@ -390,6 +390,26 @@ def values(self) -> typing.ValuesView[str]:
 
 DEFAULT_TRACE_STATE = TraceState.get_default()
 
+_TRACE_ID_BYTES = 16
```

Comment:
Please refrain from declaring single-use constants.

Reply:
Removed const variables and `_validate_trace_id`, and used `2 ** 128 - 1` as the limit checker.